### PR TITLE
De 1129 de-webhooks table, constraints  and data

### PR DESCRIPTION
**Pull request by sri2k1us**
# Tables
- `webhooks_type` A table to store the type of webhooks supported by DE. For now its just `Slack`
- `webhooks_topic` A table to store the type of notification topics that an user can subscribe to. For e.g: Data, Apps, Analyses
- `webhooks` A table to store users webhook information
- `webhooks_subscription` A tables stores users subscription to various topics for a given webhook

### CODE REVIEW

**Comment by tedgin** [comment]
LGTM :shipit:

File: src/main/constraints/78_webhooks.sql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+ALTER TABLE ONLY webhooks
+      ADD  CONSTRAINT user_id_fkey
+      FOREIGN KEY (user_id)
+      REFERENCES users(id);
+
+--
+-- Foreign key constraint for the type_id field of the webhooks table.
+--
+ALTER TABLE ONLY webhooks
+      ADD  CONSTRAINT type_id_fkey
```
Comment (psarando) [inline src/main/constraints/78_webhooks.sql]: I thought `CONSTRAINT` names had to be globally unique in the schema (so not just unique for the `webhooks` table). So maybe these constraints could be less generic? Maybe `webhooks_user_id_fkey` and `webhooks_type_id_fkey`?

File: src/main/tables/81_webhooks_subscription.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+SET search_path = public, pg_catalog;
+
+--
+-- webhooks_user_subscription table
```
Comment (psarando) [inline src/main/tables/81_webhooks_subscription.sql]: The table is named `webhooks_subscription`, but this comment says otherwise 😜

**Review by psarando — APPROVED** [review]
LGTM 👍 
I just had a couple of documentation nitpicks to consider.

File: src/main/constraints/78_webhooks.sql (hunk 1 above)
Comment (sri2k1us) [reply src/main/constraints/78_webhooks.sql]: 👍

File: src/main/tables/81_webhooks_subscription.sql (hunk 2 above)
Comment (sri2k1us) [reply src/main/tables/81_webhooks_subscription.sql]: Grr. Renamed the table but forgot to update comments :(

**Comment by psarando** [comment]
:shipit:

**Pull request merged**